ffi/src/bindings.h

```diff
@@ -1,17 +1,26 @@
-#include <rte_mbuf.h>
-#include <rte_mempool.h>
+/// known issues:
+// 1. https://github.com/rust-lang/rust/issues/54341
+
+// all the necessary DPDK functions, types and constants are defined
+// in the following header files.
+#include <rte_eal.h>
+#include <rte_errno.h>
+#include <rte_ethdev.h>
+#include <rte_kni.h>
+#include <rte_lcore.h>
```

Comment:
we still need the lcore include here? double-checking.

Reply:
no we don't.